### DISCUSSION TITLE
String concatination with table driven

### DISCUSSION THREAD

**Discussion by kga676** [getgauge/gauge#2128] [General]
Sample from documentation wors well
# Create projects

    |id| name      |
    |--|-----------|
    |1 | Alice     |
    |2 | Bob       |
    |3 | Eve       |

## First scenario
`* Say "hello" to <name>.`

## Second scenario
`* Say "namaste" to <name>.`
BUT If I'll print like 
`* Say "hello "+<name> to <name>.`
It will be marked as unknowm implemetation

Is there any way to use table driven plaseholder as a part of string value in step?

**Comment by sriv** [comment]
No, the spec files do not support string operations like what you have mentioned.

**Reply by kga676** [reply]
Ok, thanks for your answer
